agent/functional_tests/testdata/taskdefinitions/security-opts-nonewprivs/task-definition.json

```diff
@@ -0,0 +1,11 @@
+{
+  "family": "ecsftest-security-opts-nonewprivs",
+  "containerDefinitions": [{
+    "image": "127.0.0.1:51670/busybox:latest",
+    "name": "nonewprivs",
+    "portBindings": [],
+    "memory": 50,
+    "dockerSecurityOptions": ["no-new-privileges"],
+    "command": ["sh", "-c", "exit 0"]
```

Comment:
Yes using something similar to http://www.projectatomic.io/blog/2016/03/no-new-privs-docker/